README.md

```diff
@@ -1,24 +1,36 @@
 # Freenas-Quicksync
-###### How to guide for getting Intel Quicksync working on Freenas
-While researching this, everything I found led me to believe that Intel Quicksync would not work until we got FreeBSD 12. After researching forums online, I have gotten it to work on 11.3. I am not much more than a noob that is only tinkering but this works for me. I am using a i7 4790K and using Quicksync for 1 1080p to 720p or 1080p to 1080p stream leaves my cpu usage at 1%. I have not tested much more yet.
+##### How to guide for getting Intel Quicksync working on Freenas
+While researching this, everything I found led me to believe that Intel Quicksync would not work until we got FreeBSD 12. After researching forums online, I have gotten it to work on 11.3. I am not much more than a noob that is only tinkering but this works for me.
+## Test Results
+- I am using a i7 4790K and using Quicksync for 1 1080p to 720p or 1080p to 1080p stream leaves my cpu usage at 1%.
+- Core 4th Gen (Haswell) added H.265 and VC1 support in QuickSync. For 1080p transcode, "Haswell or better" is reasonable guidance.
```

Comment:
Lets do away with the I from now on...
There are multiple tests and PR's by multiple people now :P

Reply:
Done.